README.rst

```diff
@@ -71,26 +61,88 @@ If you want to contribute, install your cloned fork:
 Setup (integrate in an existing django project)
 -----------------------------------------------
 
-Add ``openwisp_network_topology`` to ``INSTALLED_APPS``:
+``INSTALLED_APPS`` in ``settings.py`` should look like the following (order is important):
 
 .. code-block:: python
 
     INSTALLED_APPS = [
-        # other apps
+        'django.contrib.auth',
+        'django.contrib.contenttypes',
+        'django.contrib.sessions',
+        'django.contrib.messages',
+        'django.contrib.staticfiles',
+        # all-auth
+        'django.contrib.sites',
+        'allauth',
+        'allauth.account',
+        'allauth.socialaccount',
+        # openwisp2 modules
+        'openwisp_users',
         'openwisp_network_topology',
+        # admin
+        'openwisp_network_topology.admin_theme',
```

Comment:
Oh. Well, I thought we'd be adding it too. As there is a PR to add admin theme. #8